### PR TITLE
updated boilerplate

### PR DESCRIPTION
**Pull request by NgoMike**
Made adjustments per @miken619 previous PR review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,14 @@
...
+
+## TODO:
+
+Add project description
 
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `npm start`
```
Comment (miken619) [inline README.md]:
```suggestion
`npm start`
```

File: README.md [hunk 2]
```diff
@@ -1,68 +1,14 @@
...
 
-Runs the app in the development mode.<br>
+Runs the app in the development mode.
```
Comment (miken619) [inline README.md]:
```suggestion
To run the app in development mode.
```

File: README.md [hunk 3]
```diff
@@ -1,68 +1,14 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# HRX Portal
+
+## TODO:
+
+Add project description
```
Comment (miken619) [inline README.md]:
```suggestion
[//]: <> ( TODO: Add project description.)
```

`TODO` are usually commented out and should only be visible to the developers.

File: src/pages/Home/Home.css [hunk 4]
```diff
@@ -0,0 +1 @@
+/* TODO: Add CSS specific to component. */
```
Comment (miken619) [inline src/pages/Home/Home.css]:
```suggestion
// TODO: Add styling.
```

Nit. You're using the multiline syntax for a single line comment. 

This file is already in same directory as the component, I don't think you need to mentioned the CSS will be specific to the component. Same as your other files.

File: src/pages/Home/Home.test.js [hunk 5]
```diff
@@ -0,0 +1 @@
+/* TODO: Add tests specific to component. */
```
Comment (miken619) [inline src/pages/Home/Home.test.js]:
```suggestion
// TODO: Add tests.
```

File: src/pages/Login/Login.css [hunk 6]
```diff
@@ -0,0 +1 @@
+/* TODO: Add CSS specific to component. */
```
Comment (miken619) [inline src/pages/Login/Login.css]:
```suggestion
// TODO: Add styling.
```

File: src/pages/Login/Login.test.js [hunk 7]
```diff
@@ -0,0 +1 @@
+/* TODO: Add tests specific to component. */
```
Comment (miken619) [inline src/pages/Login/Login.test.js]:
```suggestion
// TODO: Add tests.
```

File: src/pages/NotFoundPage/NotFoundPage.css [hunk 8]
```diff
@@ -0,0 +1 @@
+/* TODO: Add CSS specific to component. */
```
Comment (miken619) [inline src/pages/NotFoundPage/NotFoundPage.css]:
```suggestion
// TODO: Add styling.
```

File: src/pages/NotFoundPage/NotFoundPage.test.js [hunk 9]
```diff
@@ -0,0 +1 @@
+/* TODO: Add tests specific to component. */
```
Comment (miken619) [inline src/pages/NotFoundPage/NotFoundPage.test.js]:
```suggestion
// TODO: Add tests.
```

**Comment by mishafrenkel** [comment]
@shockey we need a few more approvals or someone authorized to merge to close this PR. Looks like I am do not have permissions. This is a trivial PR. Looks like Michael updated the boilerplate files with //TO DO Comments because certain files in the src/pages directory were empty.

**Comment by NgoMike** [comment]
@shockey can you merge this? thanks

**Pull request merged**